iidm/iidm-extensions/src/main/java/com/powsybl/iidm/network/extensions/ManualFrequencyRestorationReserve.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.network.extensions;
+
+import com.powsybl.commons.extensions.Extension;
+import com.powsybl.iidm.network.Generator;
+
+/**
+ * @author Jacques Borsenberger {@literal <jacques.borsenberger at rte-france.com}>
```

Comment:
The final `}` and `>` are inverted:

```suggestion
 * @author Jacques Borsenberger {@literal <jacques.borsenberger at rte-france.com>}
```